packages/client/src/testing/fixtures.ts

```diff
@@ -178,6 +180,33 @@ export function flagMock({
   }
 }
 
+export function flagPresentableMock({
```

Comment:
Nice! But it looks like need to set the `isStarred` properties in the result data.